src/test/java/org/candlepin/subscriptions/capacity/CapacityReconciliationControllerTest.java

```diff
@@ -54,189 +58,100 @@
 class CapacityReconciliationControllerTest {
```

Comment:
I refactered this test, so we can have shorter unit tests and reuse the statements.